assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTABanner.js

```diff
@@ -106,10 +109,15 @@ function AudienceSegmentationSetupCTABanner( { id, Notification } ) {
 		setShowErrorModal( true );
 	}, [ setShowErrorModal ] );
 
+	const onOAuthNavigation = useCallback( async () => {
+		await pinNotification( id, NOTIFICATION_GROUPS.SETUP_CTAS );
+	}, [ id, pinNotification ] );
```

Comment:
We can simply return the result of `pinNotification()` here, the returned Promise will be awaited by the caller:

```suggestion
	const onOAuthNavigation = useCallback(
		() => pinNotification( id, NOTIFICATION_GROUPS.SETUP_CTAS ),
		[ id, pinNotification ]
	);
```